README.md

```diff
@@ -1,42 +1,133 @@
-![openui5](http://openui5.org/images/OpenUI5_new_big_side.png)
+![OpenUI5 logo](http://openui5.org/images/OpenUI5_new_big_side.png)
 
 # openui5-worklist-app
+> [OpenUI5](https://github.com/SAP/openui5) worklist app using the [UI5 Build and Development Tooling](https://github.com/SAP/ui5-tooling).
```

Comment:
after this line please add the more specific text from the old readme:
This template implements a typical worklist floorplan, one of the design patterns that is specified by the [SAP Fiori Design Guidelines](https://experience.sap.com/fiori-design/). It includes generic application functionality and tests that can be easily extended. You can find more documentation on this template [here](https://openui5.hana.ondemand.com/#docs/guide/a77f2d29299247f8a3e30226507b1765.html).